packages/react/src/reactrouterv4v5/integration.ts

```diff
@@ -0,0 +1,158 @@
+import { WINDOW, startBrowserTracingNavigationSpan, startBrowserTracingPageLoadSpan } from '@sentry/browser';
+import {
+  SEMANTIC_ATTRIBUTE_SENTRY_OP,
+  SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN,
+  SEMANTIC_ATTRIBUTE_SENTRY_SOURCE,
+  defineIntegration,
+} from '@sentry/core';
+import type { Client, IntegrationFn, TransactionSource } from '@sentry/types';
+import { logger } from '@sentry/utils';
+import { DEBUG_BUILD } from '../debug-build';
+import { V4_SETUP_CLIENTS, V5_SETUP_CLIENTS } from './global-flags';
+import { matchRoutes } from './route-utils';
+import type { MatchPath, RouteConfig, RouterHistory } from './types';
+
+const INTEGRATION_NAME_V4 = 'ReactRouterV4';
+
+const INTEGRATION_NAME_V5 = 'ReactRouterV5';
+
+interface DefaultReactRouterOptions {
+  /**
+   * The history object from `createBrowserHistory` (or equivalent).
+   */
+  history: RouterHistory;
+}
+
+interface RouteConfigReactRouterOptions extends DefaultReactRouterOptions {
+  /**
+   * An array of route configs as per the `react-router-config` library
+   */
+  routes: RouteConfig[];
+  /**
+   * The `matchPath` function from the `react-router` library
+   */
+  matchPath: MatchPath;
+}
+
+/**
+ * Options for React Router v4 and v4 integration
+ */
+type ReactRouterOptions = DefaultReactRouterOptions | RouteConfigReactRouterOptions;
+
+// @ts-expect-error Don't type narrow on routes or matchPath to save on bundle size
+const _reactRouterV4 = (({ history, routes, matchPath }: ReactRouterOptions) => {
```

Comment:
So the main thing I see here is - or maybe I am missing this somehow - that we do not disable the default page load/navigation spans emitted by the default `browserTracingIntegration()` here 🤔 So this would be emitted twice when a router integration is added, wouldn't it?